README.md

```diff
@@ -30,12 +30,23 @@ Requirements:
 
 - go 1.6 or higher
 - A [working golang](https://golang.org/doc/code.html) environment
+- A [properly configured](https://golang.org/doc/code.html#Organization) Go workspace
 
-From the project root directory, run:
+In your terminal, run:
 ```
+cd $GOPATH
```

Comment:
I did add a link above that explains how the proper Go workspace should be setup.